scala-collector/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM java:8-alpine
+
+ENV SCALA_COLLECTOR=20170611 \
+BUILD_PACKAGES='bash curl-dev git curl ca-certificates gettext' \
+SBT_VERSION=0.13.15 \
+SBT_HOME=/usr/local/sbt \
+PATH=${PATH}:${SBT_HOME}/bin
+
+
+RUN apk update && apk upgrade && \
+apk add --update $BUILD_PACKAGES && \
+rm -rf /var/cache/apk/* && \
+
+# Install sbt
+echo 'curling' && \
+curl -sL "https://github.com/sbt/sbt/releases/download/v$SBT_VERSION/sbt-$SBT_VERSION.tgz" | gunzip | tar -xv -C /usr/local && \
+echo -ne "- with sbt $SBT_VERSION\n" >> /root/.built &&\
+apk del curl && \
+
+mkdir -p /usr/local && cd /usr/local/ && git clone https://github.com/snowplow/snowplow.git && \
+cd snowplow/2-collectors/scala-stream-collector && \
+/usr/local/sbt/bin/sbt assembly && \
+cd /usr/local/snowplow/2-collectors/scala-stream-collector/target/scala-2.10 && \
+mkdir -p /usr/local/scalacollector/ && \
```

Comment:
would it be possible to rename this folder (and the repo folder too `scala-stream-collector/Dockerfile`) `scala-stream-collector`?